frontend/views/containers/modals/JoinGroup.vue

```diff
@@ -0,0 +1,216 @@
+
+      .slide(v-observer:1='updateIndicator' :id='config[1]')
+        .slide-img
+          svg-proposal
+
+        i18n(tag='h4') 2. Wait for the group vote
+        i18n(tag='p') On Group Income, every major decision goes through a voting process. This includes adding new members.
+
+      .slide(v-observer:2='updateIndicator' :id='config[2]')
+        .slide-img
+          svg-access
+
+        i18n(tag='h4') 3. Use your unique access link
+        i18n(tag='p') Once the group agrees that you should join them, a unique access link will be generated, giving you instant access to the group
+
+    .dots
+      a.dot(v-for='(link, i) in config' :class='{ active: indicator == i }' :href='"#"+link')
+</template>
+
+<script>
+import ModalBaseTemplate from '@components/Modal/ModalBaseTemplate.vue'
+import { REPLACE_MODAL } from '@utils/events.js'
+import sbp from '~/shared/sbp.js'
+import SvgAccess from '@svgs/access.svg'
+import SvgInvitation from '@svgs/invitation.svg'
+import SvgProposal from '@svgs/proposal.svg'
+
+export default {
+  name: 'JoinGroupModal',
+  components: {
+    ModalBaseTemplate,
+    SvgAccess,
+    SvgInvitation,
+    SvgProposal
+  },
+  data () {
+    return {
+      indicator: 0,
+      config: [
+        'get-an-invitation',
+        'wait-for-you-group-vote',
+        'use-your-unique-access-link'
+      ]
+    }
+  },
+  // TODO move this into slider component if we have more than one
+  directives: {
+    observer: {
+      inserted: (el, { value, arg }) => {
+        try {
+          const io = new window.IntersectionObserver(
+            (elements) => {
+              if (elements[0].intersectionRatio >= 0.5) { // Wait for the middle of the screen
+                value(arg)
+              }
+            },
+            { threshold: [0.5] }
+          )
+
+          io.observe(el)
+
+          el.stop = () => {
+            io.disconnect()
+          }
+        } catch (error) {
+          console.error(error.message, '\n polyfill: https://github.com/w3c/IntersectionObserver')
+        }
+      }
+    },
+    unbind (el) {
+      el.stop()
+    }
+  },
+  methods: {
+    showLoginModal () {
+      sbp('okTurtles.events/emit', REPLACE_MODAL, 'CreateGroup')
+    },
+    updateIndicator (i) {
+      this.indicator = i
+      history.pushState(null, null, `#${this.config[i]}`)
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "../../../assets/style/_variables.scss";
+
+.modal {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: $general_2;
+}
+
+.wrapper {
+  width: 100%;
+  margin: 0 auto;
+
+  @include desktop {
+    width: auto;
+  }
+}
+
+.slider {
+  display: flex;
+  overflow-x: auto;
+  -ms-overflow-style: none;
+  overflow: -moz-scrollbars-none;
+
+  /* make it smooth on iOS */
+  -webkit-overflow-scrolling: touch;
+  scroll-snap-points-x: repeat(100vw);
+  scroll-snap-type: x mandatory;
+
+  &::-webkit-scrollbar {
+    width: 0 !important;
```

Comment:
This doesn't hide the scroll bar on Safari and Firefox. The only effective trick to hide I know is to use a pseudo-element above the scrollbar with the same color as the background to hide it.

Reply:
It's working in safari. Only firefox mobile was not working. It's fixed now. Thanks for the heads up.